### PR TITLE
refactored contract functions out into sep file

### PR DESCRIPTION
**Pull request by W3stside**
Refactored out contract code into sep file

### CODE REVIEW

**Comment by josojo** [comment]
I think the fact that the third test command is not going through has nothing to do with your specific implementation. Had these issues before. Maybe accounts ran out of funding. Idk.
  
I like the refactoring a LOT

File: test/testFunctions.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  'TokenTUL',
+  'PriceOracle',
+]
+
+/**
+ * >getContract()
+ * deploys all contracts in build/contracts
+ */
+const getContracts = async () => {
+  const depContracts = contractNames.map(c => artifacts.require(c)).map(async cc => cc.deployed())
```
Comment (Velenir) [inline test/testFunctions.js]: `async` is unnecessary here

File: test/testFunctions.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+const setupTest = async (accounts, { DutchExchange: dx, EtherToken: eth, TokenGNO: gno, PriceOracle: oracle }) => {
+  // Await ALL Promises for each account setup
+  await Promise.all(accounts.map((acct) => {
+    /* eslint array-callback-return:0 */
+    if (acct === accounts[0]) return
+
+    eth.deposit({ from: acct, value: 10 ** 9 })
+    eth.approve(dx.address, 10 ** 9, { from: acct })
+    gno.transfer(acct, 10 ** 18, { from: accounts[0] })
+    gno.approve(dx.address, 10 ** 18, { from: acct })
```
Comment (Velenir) [inline test/testFunctions.js:47]: `await Promise.all` on line 38 doesn't ultimately wait for the `eth.* gno.*` transactions, but only starts them. `accounts.map` maps to `Array<undefined>` not promises. Thanks to the sequential nature of `testrpc`'s transaction handling, it won't usually break anything, but the right thing would be to wrap them in `Promise.all()`:

```
return Promise.all([eth.deposit, eth.approve, gno.transfer, gno.approve])
```

Same with lines 51, 52 below.

File: test/testFunctions.js (hunk 1 above)
Comment (W3stside) [reply test/testFunctions.js]: yep no idea why thats in there O.O

File: test/testFunctions.js (hunk 2 above)
Comment (W3stside) [reply test/testFunctions.js:47]: true true

**Pull request merged**